.jenkinsci/debug-build.groovy

```diff
@@ -45,74 +45,71 @@ def doDebugBuild(coverageEnabled=false) {
     }
   }
 
-  docker.image('postgres:9.5').withRun(""
-    + " -e POSTGRES_USER=${env.IROHA_POSTGRES_USER}"
-    + " -e POSTGRES_PASSWORD=${env.IROHA_POSTGRES_PASSWORD}"
-    + " --name ${env.IROHA_POSTGRES_HOST}"
-    + " --network=${env.IROHA_NETWORK}") {
-      iC.inside(""
-      + " -e IROHA_POSTGRES_HOST=${env.IROHA_POSTGRES_HOST}"
-      + " -e IROHA_POSTGRES_PORT=${env.IROHA_POSTGRES_PORT}"
-      + " -e IROHA_POSTGRES_USER=${env.IROHA_POSTGRES_USER}"
-      + " -e IROHA_POSTGRES_PASSWORD=${env.IROHA_POSTGRES_PASSWORD}"
-      + " --network=${env.IROHA_NETWORK}"
-      + " -v /var/jenkins/ccache:${CCACHE_DIR}"
-      + " -v /tmp/${GIT_COMMIT}-${BUILD_NUMBER}:/tmp/${GIT_COMMIT}") {
+  sh "docker run -td -e POSTGRES_USER=${env.IROHA_POSTGRES_USER} \
+  -e POSTGRES_PASSWORD=${env.IROHA_POSTGRES_PASSWORD} --name ${env.IROHA_POSTGRES_HOST} \
+  --network=${env.IROHA_NETWORK} postgres:9.5 -c 'max_prepared_transactions=100'"
```

Comment:
Pls, add an explanation of `max_prepared_transactions` and reason to put it as `100`.